src/graphql/schema/publication.js

```diff
@@ -0,0 +1,9 @@
+export default `
+  type Publication {
+    id: ID!
+    name: String!
+    image: String!
+    enabled: Boolean
+    twitter: String
```

Comment:
This field is better be removed for now

Reply:
This is related to the comment above :D.

Reply:
I mean specifically the `twitter` field